subsys/spm/spm.c

```diff
@@ -355,7 +360,7 @@ static void spm_configure_ns(const tz_nonsecure_setup_conf_t
 	/* Set non-banked exceptions to target Non-Secure */
 	tz_nbanked_exception_target_state_set(0);
 	/* Don't allow Non-Secure firmware to issue System resets. */
```

Comment:
Update comment to reflect the configurability

Reply:
Think this is addressed now